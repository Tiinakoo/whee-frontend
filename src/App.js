import "./App.css";

function App() {
  return <div className="App">This will be Whee-store</div>;
}

export default App;
