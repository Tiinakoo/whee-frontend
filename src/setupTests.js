import Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import "jest-enzyme";
import React from "react";

Enzyme.configure({ adapter: new Adapter() });

React.useLayoutEffect = React.useEffect;
