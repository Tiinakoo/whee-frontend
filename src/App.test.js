import App from "./App";
import { mount } from "enzyme";
import React from "react";
import "./setupTests";

describe("App", () => {
  let component;
  let shapesModelStub;

  beforeEach(() => {
    shapesModelStub = {
      shapes: [
        {
          id: "some-shape-id",
          name: "some-shape-name",
          description: "some-shape-description",
          price: 100,
          currency: "â‚¬",
          addToCart: expect.any(Function),
        },
      ],
    };

    component = mount(<App shapesModel={shapesModelStub} />);
  });

  it("renders", () => {
    expect(component).toMatchSnapshot();
  });
});
