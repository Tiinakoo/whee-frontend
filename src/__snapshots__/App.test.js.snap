// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<div
  className="App"
>
  This will be Whee-store
</div>
`;
